core/runtime/src/main/java/io/quarkus/runtime/logging/LoggingSetupRecorder.java

```diff
@@ -104,8 +104,9 @@ public ShutdownListener initializeLogging(LogConfig config, LogBuildTimeConfig b
         final Logger rootLogger = logContext.getLogger("");
 
         if (config.level.intValue() < buildConfig.minLevel.intValue()) {
-            log.warnf("Root log level %s set below minimum logging level %s, promoting it to %s",
-                    config.level, buildConfig.minLevel, buildConfig.minLevel);
+            log.warnf(
+                    "Root log level %s set below minimum logging level %s, promoting it to %s. Set the build time configuration property 'quarkus.log.min-level' to '%s' to avoid this warning",
```

Comment:
Actually a copy-pastable would be nicer: 
```suggestion
                    "Root log level %s set below minimum logging level %s, promoting it to %s. Set the build time configuration property 'quarkus.log.min-level=%s' to avoid this warning",
```

Reply:
The problem with the copy pasteable configuration is that it only works for `application.properties`, no other config source

Reply:
Right, but at least it works for one (which I believe it's mostly used) :)

But I don't mind, it's fine if you want to leave as is

Reply:
👌🏼